packages/main/src/Table.ts

```diff
@@ -566,6 +564,10 @@ class Table extends UI5Element {
 		this.fireDecoratorEvent("row-action-click", { action, row });
 	}
 
+	_$<T = HTMLElement>(selector: string): T {
+		return this.shadowRoot?.querySelector(selector) as T;
```

Comment:
You can check `query` and `queryAll` decorators which are doing something similar.

Reply:
that is cool!! good to know thx